### DISCUSSION TITLE
Better UI resizing & Keyboard Support in Windows

### DISCUSSION THREAD

**Discussion by mathiaswillburger** [beyondcode/herd-community#717] [Ideas]
It would be great to have more flexibility in resizing the UI in the Windows version. The sidebars and containers in the mail window and log viewer are fixed so that longer titles in subjects or log files with timestamps are cut off.

It would also be great to have Keyboard support in the Mails view, i.e., allowing deleting emails with DEL, navigating with the arrow keys, selecting multiple emails for deletion, etc.

The main window also only has a fixed layout, so in the MySQL log view, content is cut off immediately and there is no way to avoid immediate scrolling.

![2024-05-08_12-07-51](https://github.com/beyondcode/herd-community/assets/28543450/0cf9532b-8c8d-4ad9-86b5-788c57160295)
![2024-05-08_12-15-46](https://github.com/beyondcode/herd-community/assets/28543450/765fcaf0-a583-4a8b-8035-c5e164e39001)

**Comment by mathiaswillburger** [comment]
Adding to this would be amazing:
- resizing possibilities for font sizes in Dumps view
- and dark mode for dumps view for better readability of output

**Reply by SerjoA** [reply]
also for mac, the ability to view dumps in herd pro is useless, very small font
